packages/docs/src/examples/va-divider/Vertical.vue

```diff
@@ -1,9 +1,15 @@
 <template>
   <div class="row my-5">
-    <span class="flex lg1">Title</span>
```

Comment:
Could be moved to generic translations to simplify reuse.
mb fillerText or something.

Reply:
moved to generic translations